src/barco.c

```diff
@@ -44,8 +49,8 @@ int main(int argc, char **argv) {
                      "directory to mount as root in the container"),
       cmd =
           arg_strn("c", "cmd", "<s>", 1, 1, "command to run in the container"),
-      arg =
-          arg_strn("a", "arg", "<s>", 0, 1, "argument to pass to the command"),
+      arg = arg_strn("a", "arg", "<s>", 0, _POSIX_ARG_MAX,
```

Comment:
Thanks for defining the constant. I am now thinking:

- It will probably conflict with `ARGTABLE_ARG_MAX`, is it something you checked?
- `_POSIX_ARG_MAX` is probably defined in some Linux header already
- `ARG_MAX` might be  a better choice, it is defined in [limits.h](https://pubs.opengroup.org/onlinepubs/7908799/xsh/limits.h.html)

Reply:
> It will probably conflict with ARGTABLE_ARG_MAX, is it something you checked?
I just tested, I passed 100 `-a -something` seems no conflict here.
> ARG_MAX might be a better choice, it is defined in [limits.h](https://pubs.opengroup.org/onlinepubs/7908799/xsh/limits.h.html)
Thanks for your help, I tried to search it but I didn't find it. I'm totally unfamiliar with C

Reply:
I got you, but I don't know how to write compatible for mutil-arch, I guess just like:

```
#if defined(LINUX_BABAB)
#include <sys/linux/limits.h>
#elif defined(BALABALA)
#include <sys/limits.h>
#else
#include <linux/limits.h>
```

I have no idea how to get these conditions.

Reply:
Ah, I wouldn't worry about multi-arch, that's not something I have ever considered. Whatever header works on Linux v6.1.0 on Debian 12 (or in CI really) is fine by me. If I recall correctly, I used `#include <sys/limits.h>` elsewhere too.